package.json

```diff
@@ -17,7 +17,8 @@
     "dev-watch-es5": "npm run gulp watch_ru_all_es5 gen-service-worker-es5",
     "lint_js": "eslint src panels js --ext js,html",
     "lint_html": "ls -1 src/home-assistant.html panels/**/ha-panel-*.html | xargs polymer lint --input",
-    "test": "npm run lint_js && npm run lint_html"
+    "mocha": "BABEL_ENV=mocha node_modules/.bin/mocha --recursive --compilers js:babel-core/register test-mocha",
```

Comment:
Let's create a file called `test-mocha/mocha.opts` for the command line flags, one per line:

```
--recursive
--compilers js:babel-core/register
--timeout 10000
```